tests/tools/test_parmetis_wrapper.py

```diff
@@ -0,0 +1,271 @@
+import argparse
+import dgl
+import json
+import numpy as np
+import os
+import sys
+import tempfile
+import torch
+
+from dgl.data.utils import load_tensors, load_graphs
+
+from chunk_graph import chunk_graph
+
+def test_parmetis_wrapper():
```

Comment:
pls add tests for `train/val/test_mask`...

Reply:
Added test as per suggestion.